src/Symfony/Component/Serializer/Tests/Normalizer/DateTimeNormalizerTest.php

```diff
@@ -206,6 +206,19 @@ public function testDenormalize()
     public function testDenormalizeUsingTimezonePassedInConstructor()
     {
         $this->doTestDenormalizeUsingTimezonePassedInConstructor();
+
+        // Test correction of Timezone when timezone information is added in both the date string and the context of the normalizer.
```

Comment:
Maybe this could be directly in a new test ?

Reply:
@Simperfit Sure, can do.  It would be good to have a why you think this, given that the test name is spot on.

Reply:
I think it already test a case, and you are fixing another case.
Within this test you are testing 2 case IMHO.

Reply:
Thanks.  All done as suggested, using ...InDefaultContext

Reply:
Time zone or timezone? You use both variants in the same sentence